ppb/engine.py

```diff
@@ -262,6 +333,14 @@ def flush_events(self):
         self.events = deque()
 
     def walk(self):
+        """
+        Walk the object tree.
+
+        Publication order: The :class:`GameEngine`, the :class:`GameEngine`
```

Comment:
You list GameEngine twice

Reply:
That's "The GameEngine System list". It probably reads fine without it, though.